tool/tctl/common/resource_command.go

```diff
@@ -754,6 +758,28 @@ func (rc *ResourceCommand) createSAMLIdPServiceProvider(ctx context.Context, cli
 	return nil
 }
 
+func (rc *ResourceCommand) createDevice(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
+	if rc.IsForced() {
+		return trace.BadParameter("Devices cannot be overwritten with the --force flag.")
```

Comment:
WDYT about printing a warning but calling the RPC anyway? The --force may be harmless if the device doesn't exist.

Reply:
Sure. Addressed